script/verification/IResourceMetering.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+interface IResourceMetering {
+    struct ResourceParams {
+        uint128 prevBaseFee;
+        uint64 prevBoughtGas;
+        uint64 prevBlockNum;
+    }
+
+    struct ResourceConfig {
+        uint32 maxResourceLimit;
+        uint8 elasticityMultiplier;
+        uint8 baseFeeMaxChangeDenominator;
+        uint32 minimumBaseFee;
+        uint32 systemTxMaxGas;
+        uint128 maximumBaseFee;
+    }
+
+    error OutOfGas();
+
+    event Initialized(uint8 version);
+
+    function params() external view returns (uint128 prevBaseFee, uint64 prevBoughtGas, uint64 prevBlockNum); // nosemgrep
+
+    function __constructor__() external;
```

Comment:
NIT: curious to know if this is required?

Reply:
[cd7ec86](https://github.com/ethereum-optimism/superchain-ops/pull/418/commits/cd7ec86dcb90083996058af5ca57e1233c4374cf)